src/sentry/api/bases/organization.py

```diff
@@ -430,7 +433,11 @@ def get_filter_params(
         try:
             if isinstance(project_ids, list):
                 project_ids = set(project_ids)
-            projects = self.get_projects(request, organization, project_ids=project_ids)
+            if isinstance(project_slugs, list):
+                project_slugs = set(project_slugs)
+            projects = self.get_projects(
```

Comment:
It seems like if `project_ids` are provided the parent class completely ignores the slugs. Is that what you intended to do?

Reply:
Yea - correct

This is slightly tangential to this actual PR.
I can move this into a separate PR.

but essentially yes- this is what was happening today for project_ids.
I added another arg for `project_slugs` which follows the existing pattern.

the TLDR is that the helper will default to fetching the passed projects rather than pulling them from the request query if they're passed in.

Reply:
Split the PR out here: https://github.com/getsentry/sentry/pull/62940